### DISCUSSION TITLE
百度网盘一直无法加载

### DISCUSSION THREAD

**Discussion by KimmyLueng** [AlistGo/alist#2456] [Q&A]
failed get objs: failed to list objs: Get "https://pan.baidu.com/rest/2.0/xpan/file?*************************************************************************************************dir=%2F&limit=200&method=list&order=name&start=0&web=web": net/http: TLS handshake timeout

我部署在vps上的

**Comment by HsinKwok** [comment]
百度盘挂上不限速吗？

**Reply by KimmyLueng** [reply]
在哪里设置？之前alist都是部署在本地的

**Reply by HsinKwok** [reply]
我还没挂百度盘，存的东西太多太杂

**Comment by KimmyLueng** [comment]
一刻相册就没有问题

**Comment by KimmyLueng** [comment]
貌似是因为vps在国外，访问受限

**Reply by Muggles001** [reply]
解决了吗老哥，我也这个毛病

**Reply by Maogxer** [reply]
我的也是，刚挂上的时候是正常的，过了一天就不行了。
